{
	"candidate": "CVE-2015-5143",
	"description": "The session backends in Django before 1.4.21, 1.5.x through 1.6.x, 1.7.x before 1.7.9, and 1.8.x before 1.8.3 allows remote attackers to cause a denial of service (session store consumption) via multiple requests with unique session keys.",
	"priority": "medium",
	"references": [
		"https://www.djangoproject.com/weblog/2015/jul/08/security-releases/",
		"https://ubuntu.com/security/notices/USN-2671-1",
		"https://www.cve.org/CVERecord?id=CVE-2015-5143"
	],
	"assigned_to": "mdeslaur",
	"discovered_by": "Eric Peterson and Lin Hua Cheng",
	"public_date": "2015-07-14 17:59:00 UTC",
	"public_date_at_usn": "2015-07-08",
	"crd": "2015-07-08",
	"packages": {
		"devel": {
			"python-django": {
				"name": "python-django",
				"status": "released",
				"note": "1.7.9-1ubuntu1"
			}
		},
		"esm-infra-legacy/trusty": {
			"python-django": {
				"name": "python-django",
				"status": "not-affected",
				"note": "1.6.1-2ubuntu0.9"
			}
		},
		"precise": {
			"python-django": {
				"name": "python-django",
				"status": "released",
				"note": "1.3.1-4ubuntu1.17"
			}
		},
		"trusty": {
			"python-django": {
				"name": "python-django",
				"status": "released",
				"note": "1.6.1-2ubuntu0.9"
			}
		},
		"trusty/esm": {
			"python-django": {
				"name": "python-django",
				"status": "not-affected",
				"note": "1.6.1-2ubuntu0.9"
			}
		},
		"upstream": {
			"python-django": {
				"name": "python-django",
				"status": "released",
				"note": "1.4.21,1.7.9,1.8.3"
			}
		},
		"utopic": {
			"python-django": {
				"name": "python-django",
				"status": "released",
				"note": "1.6.6-1ubuntu2.3"
			}
		},
		"vivid": {
			"python-django": {
				"name": "python-django",
				"status": "released",
				"note": "1.7.6-1ubuntu2.1"
			}
		}
	}
}
